6.2.2000_run.py

```diff
@@ -0,0 +1,14 @@
+import time
+
+
+def timer(f, *args, **kwargs):
+    t1 = time.time()
+    f(*args, **kwargs)

```

Comment:
rename 'f' and 't1' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing